Automattic/woocommerce-payments
# Fix error when disabling WCPay with core disabled

Fixes #7303

#### Changes proposed in this Pull Request








Fix error when disabling WCPay with core disabled.

#### Testing instructions





* Disable WooCommerce core.
* Disable WCPay.
* No errors should be thrown.

-------------------


**Post merge**
